openedx/core/djangoapps/theming/test_util.py

```diff
@@ -81,12 +70,13 @@ def with_edx_domain_context(is_edx_domain):
 
     """
     if is_edx_domain:
-        changes = comprehensive_theme_changes(EDX_THEME_DIR)
-        with override_settings(COMPREHENSIVE_THEME_DIR=EDX_THEME_DIR, **changes['settings']):
-            with edxmako.save_lookups():
-                for template_dir in changes['template_paths']:
-                    edxmako.paths.add_lookup('main', template_dir, prepend=True)
-
+        domain = 'edx.org'
```

Comment:
We shouldn't hard code edx.org into the platform. The default should be the open source theme, and then edx.org should be provided as a configuration for prod.